server/server.js

```diff
@@ -1,71 +1,74 @@
 const express = require('express');
 const app = express();
 const path = require('path');
-const PORT = 3000;
 
+// dotENV
+require('dotenv').config();
+
+// port
+const PORT = process.env.PORT || 3000;
+
+// require in our routers
 const usersRouter = require('./routes/users');
 const newsRouter = require('./routes/news');
+
+// controllers
 const friendsController = require('./controllers/friends-controller');
 
 //handle incoming requests
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 
-// app.use(express.static(path.resolve(__dirname, '../dist')));;
-
-//serves the html file
+// send static files, when requeted from / endpoint
 app.get('/', (req, res) => {
+  console.log('request to /');
   console.log('GET REQUEST RECEIVED, sending HTML file');
   res.sendFile(path.join(__dirname, '../client/index.html'));
 });
 
-// sends array of user objects when request to /users endpoint is received
+// routher for users
 app.use('/users', usersRouter);
-
-// send signup html file
-// app.get('/signup', (req, res) => {
-//     res.sendFile(path.resolve(__dirname, '../client/signup.html'));
-//   });
+// roucter for news
+app.use('/news', newsRouter);
 
 // create new user
 app.post('/signup', friendsController.createUser, (req, res) => {
-  // redirect user to their homepage
-  // res.redirect('/');
+  console.log('request to /signup');
   console.log('redirect to homepage');
   res.sendStatus(200);
 });
 
-//login
+// login
 app.post('/login', friendsController.verifyUser, (req, res) => {
-  //   res.redirect('/');
-  //  redirect user to homepage on success
-  //   console.log(res.locals.user.length);
+  console.log('request to /login');
   if (res.locals.user.length === 0) return next({ log: 'invalid login' });
   res.sendStatus(200);
 });
 
+// not sure what this is for. 
 app.post('/send', friendsController.sendMessage, (req, res) => {
+  console.log('post to /send');
   res.sendStatus(200);
 });
 
-app.use('/news', newsRouter);
+// catch all
+app.use((req, res) => res.sendStatus(404));
```

Comment:
app.use('*', (req, res, next) => {
  const err = {
    log: 'Request to unknown path',
    status: 404,
    message: { err: 'Not found' }
  }
  return next(err);
})